src/apprt/gtk/Window.zig

```diff
@@ -774,12 +775,15 @@ fn gtkRefocusTerm(v: *c.GtkWindow, ud: ?*anyopaque) callconv(.C) bool {
 
     return true;
 }
-
 fn gtkCloseRequest(v: *c.GtkWindow, ud: ?*anyopaque) callconv(.C) bool {
     _ = v;
     log.debug("window close request", .{});
     const self = userdataSelf(ud.?);
+    self.closeWithConfirmation();
+    return true;
+}
 
+pub fn closeWithConfirmation(self: *Window) void {
```

Comment:
I don't love the name of this function given the next comment. What about just close with documentation in a function level comment?